addons/mail/static/src/scss/composer.scss

```diff
@@ -159,3 +165,8 @@
         padding: 12px 18px 11px 16px;
     }
 }
+
+.o_document_viewer_topbar_button {
```

Comment:
Note: only download has this class, other buttons don't have it for some reason.

Reply:
Made change to put this classname on all buttons of the topbar of document viewer